src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/toolbars/CustomizedToolbarComponentObject.java

```diff
@@ -151,30 +151,6 @@ public void verifyFollowMessage() {
   }
 
   /**
-   * Verify that page is followed The method should be used only after clicking on "follow" button.
-   * Before that, "follow" button does not have 'title' attribute which is necessary in the method
-   */
```

Comment:
I removed the method as Watch and Unwatch buttons on toolbar don't have `title` attribute anymore (for both states, before and after click). Method was used only in `CustomizeToolbar006_ButtonsActions` test.
There is another method in test which checks if button label was correctly changed after clicking.